### DISCUSSION TITLE
"Hai specificato una ritenuta d'acconto senza specificarne la causale"

### DISCUSSION THREAD

**Discussion by ratataplan** [fattureincloud/api#32] [Q&A] [answered]
"Hai specificato una ritenuta d'acconto senza specificarne la causale" è il messaggio di errore che compare con la verifica della fattura elettronica. Eppure la causale è Impostata: A
Il documento è stato esportato correttamente dal mio applicativo Windows con le nuove API 2.0 ma la verifica formale segnala questo errore
![verifica_formale_causale_ritenuta](https://user-images.githubusercontent.com/11733873/181789954-381a8c68-f40b-403d-bbc8-861ed5855a41.jpg)
La ritenuta è impostata al 20%
se visualizzo la fattura elettronica ottengo
**Ritenuta
Tipologia ritenuta: RT01(ritenuta persone fisiche)
Importo ritenuta: 4.16
Aliquota ritenuta (%): 20.00
Causale di pagamento: A(decodifica come da modello CU)**

Quali proprietà devo impostare? Ho provato con ei_raw ma non ho nessuna proprietà che risolva il caso.

**Comment by valmoz** [comment]
Ciao @ratataplan 
se hai impostato la causale correttamente non dovrebbe succedere.
Ti chiederei di contattare il supporto clienti sulla pagina di Fatture in Cloud, in modo tale da poter richiedere le informazioni richieste che non possiamo scambiarci sulla discussione pubblica.
Puoi inserire nella richiesta il link a questa discussione per permettere al supporto di essere aggiornati a riguardo.

Grazie
Mauro

**Reply by valmoz** [reply]
@ratataplan per favore, controlla se qui vedi Causale A:
![Schermata 2022-07-29 alle 17 41 16](https://user-images.githubusercontent.com/7254293/181795571-dc5789de-1d9d-4793-bc52-4f4101351413.png)
In caso contrario, non è stata settata correttamente.

Il campo comunque è ei_withholding_tax_causal.

Mauro

**Comment by ratataplan** [comment, marked as answer]
Causale A compariva in anteprima web e anche nell'XML,
ma, mancava nel codice:
**invoice.EiWithholdingTaxCausal = "A";**
Ora la verifica formale è OK!
